sources/install/package_c2.sh

```diff
@@ -16,18 +16,13 @@ function install_metasploit() {
     # CODE-CHECK-WHITELIST=add-aliases
     colorecho "Installing Metasploit"
     fapt libpcap-dev libpq-dev zlib1g-dev libsqlite3-dev
-    mkdir /tmp/metasploit_install
-    cd /tmp/metasploit_install
-    curl https://raw.githubusercontent.com/rapid7/metasploit-omnibus/master/config/templates/metasploit-framework-wrappers/msfupdate.erb -o msfinstall
-    chmod +x msfinstall
-    rvm use 3.0.0@metasploit --create
-    ./msfinstall
-    cd /tmp
-    rm -rf /tmp/metasploit_install
-    bundle install --gemfile /opt/metasploit-framework/embedded/framework/Gemfile
-    rvm use 3.0.0@default
-    # https://github.com/ruby/fileutils/issues/22 -> Warnings
-    add-history msfconsole
```

Comment:
I agree that this line should be removed. Can you remove the `msfconsole` history file as well?